app/scripts/modules/core/src/pipeline/config/stages/findAmi/findAmiStage.module.ts

```diff
@@ -0,0 +1,9 @@
+import { module } from 'angular';
+
+import { FIND_AMI_STAGE } from './findAmiStage';
+
+export const FIND_AMI_STAGE_MODULE = 'spinnaker.core.pipeline.stage.findAmi';
+
+module(FIND_AMI_STAGE_MODULE, [
+  FIND_AMI_STAGE,
+]);
```

Comment:
I'd skip this file/module and just import the `FIND_AMI_STAGE` wherever you're importing `FIND_AMI_STAGE_MODULE`.

Reply:
There's a few of these... I'll clean them up.